logging-modules/log4j/src/main/java/com/baeldung/reactiveLogging/ReactiveLogging.java

```diff
@@ -0,0 +1,15 @@
+package com.baeldung.reactiveLogging;
+
+import reactor.core.publisher.Flux;
+
+public class ReactiveLogging {
+    
+    public static void main(String[] args) {
+        Flux<Integer> reactiveStream = Flux.range(1, 5)
```

Comment:
In your article you have multiple examples, any chance we could have all of these included in the code

Reply:
You will also notice that all of the other files in that folder are named using the convention `LibraryExample`. Can you do that in your file as well?

Reply:
Sorry for formatting - I know there is a Eclipse formatter and was sure I'm using it... but it turned out  I was not.
Everything should be fixed now.